pkg/config/config.go

```diff
@@ -54,7 +54,8 @@ type Finch struct {
 	// Requires macOS 13.0 or later and an Apple Silicon (ARM64) mac.
 	// Has no effect on systems where Rosetta 2 is not available (Intel/AMD64 macs, or macOS < 13.0).
 	// This setting will only be applied on vm init.
-	Rosetta *bool `yaml:"rosetta,omitempty"`
+	Rosetta     *bool   `yaml:"rosetta,omitempty"`
+	HostGatewayIp *string `yaml:"host_gateway_ip,omitempty"`
```

Comment:
Please add godoc comment i.e.
```suggestion
        // HostGatewayIp sets the default host gateway ip....
	HostGatewayIp *string `yaml:"host_gateway_ip,omitempty"`
```